tests/functional/generate_pages.py

```diff
@@ -409,12 +409,47 @@ def process_calculated_summary(answers, page_spec):
         page_spec.write(CALCULATED_SUMMARY_LABEL_GETTER.substitute(answer_context))
 
 
-def process_final_summary(schema_data, page_spec, collapsible, section_summary=False):
-    for section in schema_data["sections"]:
-        write_summary_spec(collapsible, page_spec, section, section_summary)
+def process_final_summary(
+    schema_data, require_path, dir_out, spec_file, collapsible, section_summary=False
+):
+    logger.debug("Processing questionnaire flow")
+
+    page_filename = f"summary.page.js"
```

Comment:
I think we need to be clear on the naming. The page is the `submit` page, that can include a (final) summary.

Reply:
Have updated.